src/pages/404.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, Flex, Image, Text, useMediaQuery } from '@chakra-ui/react';
+
+import Footer from '@/uikit/Footer';
+import Header from '@/uikit/Header';
+
+function DescriptionMobile() {
+  return (
+    <Text as='h3' fontWeight='light'>
+      Whoops, you’ve wandered far, even Pluto is in your rearview! Seems like
+      the page you’re looking for has drifted off-course
+    </Text>
+  );
+}
+
+function DescriptionDesktop() {
+  return (
+    <>
+      <Text as='h3' fontWeight='light'>
+        Whoops, you’ve wandered far, even Pluto is in your rearview!
+      </Text>
+      <Text>Seems like the page you’re looking for has drifted off-course</Text>
+    </>
+  );
+}
+
+export default function Page404() {
+  const isMobile = useMediaQuery('(max-width: 768px)')[0];
+  const isDesktop = useMediaQuery('(min-width: 1024px)')[0];
+
+  return (
+    <Flex flexDirection='column'>
+      <Box w='full' position='absolute' zIndex={15}>
+        <Header />
+      </Box>
+      <Flex
+        justifyContent='center'
+        alignItems='center'
+        flexDirection='column'
+        w='full'
+        h='100vh'
+        background='black'
+        position='relative'
+        overflow='hidden'>
+        <Flex
+          zIndex={10}
+          color='white'
+          alignItems='center'
+          flexDirection='column'
+          px='10'>
```

Comment:
To avoid confusion between value for our sizing value, I think we need to have a rules. If you want to use `number` as an value like `10` only without format, you can use object instead of string `px={10}`, you can use string format if only there is format behind the numbers `px, rem, vh, %, etc`

Reply:
updated